common/time/Cargo.toml

```diff
@@ -0,0 +1,7 @@
+[package]
```

Comment:
I'd rename this crate to `clock`. there is already a `time` crate and `clock` in more in line with what you expose (`Clock`).